app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -461,8 +460,18 @@ private fun ConversationScreenContent(
                 onShowContextMenu = onShowContextMenu
             )
         },
-        onSendTextMessage = onSendMessage,
-        onSendAttachment = onSendAttachment,
+        onSendTextMessage = { message, mentions, messageId ->
+            scope.launch {
+                lazyListState.animateScrollToItem(0)
```

Comment:
What if user is seeing some really old message? Will this animation take ages?

Reply:
Good point... I will update it to `scrollToItem` which Instantly brings the item to 0